### PR TITLE
Update bookstore tests

### PR DESCRIPTION
**Pull request by bob0the0mighty**
Updated BookStoreTest via generator. That worked as expected, but there was a failing test for the example implementation. After checking the example I realized there was a bug caused by not sorting the groupings by number of books in each group. Addressed that issue as well.

### CODE REVIEW

File: exercises/markdown/MarkdownTest.fs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// This file was auto-generated based on version 1.2.0 of the canonical data.
+// This file was auto-generated based on version 1.3.0 of the canonical data.
```
Comment (ErikSchierboom) [inline exercises/markdown/MarkdownTest.fs]: This file shouldn't be part of the PR.

File: exercises/book-store/BookStoreTest.fs [hunk 2]
```diff
@@ -11,55 +11,59 @@ open BookStore
 let ``Only a single book`` () =
     total [1] |> should equal 8.00
 
-[<Fact(Skip = "Remove to run test")>]
+[<Fact>]
```
Comment (ErikSchierboom) [inline exercises/book-store/BookStoreTest.fs]: Why have all the `(Skip = "Remove to run test")` parts been removed? Could you add them back?

**Comment by bob0the0mighty** [comment]
I've made a hash of this pull request. I didn't realize Github would continue automatically adding changes to the same branch to the pull request. 

What's the best way to resolve this?

**Comment by ErikSchierboom** [comment]
I think the simplest solution would be to copy your changed file to some other place, and then do the following on this branch:

```
git remote add exercism https://github.com/exercism/fsharp.git
git fetch --prune exercism
git reset --hard exercism/master
<copy back your changes>
<commit your changes>
git push --force
```

**Comment by bob0the0mighty** [comment]
I've made a hash of this pull request. I didn't realize Github would continue automatically adding changes to the same branch to the pull request. 

What's the best way to resolve this?
